### ISSUE TITLE
Crash (java.lang.AbstractMethodError) affecting android users (Notification?)

### ISSUE THREAD

**Issue by laurentS** [mapswipe/mapswipe#124]
Google playstore is reporting a crash affecting over 400 users between yesterday and today.

The java stacktrace looks like this:
```java
java.lang.AbstractMethodError: 
  at com.google.firebase.iid.zzb.onStartCommand (Unknown Source)
  at android.app.ActivityThread.handleServiceArgs (ActivityThread.java:4063)
  at android.app.ActivityThread.access$2400 (ActivityThread.java:222)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:1898)
  at android.os.Handler.dispatchMessage (Handler.java:102)
  at android.os.Looper.loop (Looper.java:158)
  at android.app.ActivityThread.main (ActivityThread.java:7237)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run (ZygoteInit.java:1230)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:1120)
```

It's not entirely clear what the issue is yet, but probably related to receiving a push notification (one was sent on September 30 around noon UK time). Guess based on https://stackoverflow.com/questions/39724731/abstractmethoderror-with-firebase-cloud-messaging/39725599

If so, the fix seems to rely on matching versions of different firebase dependencies, see SO answer above.

Needs investigation to reproduce and confirm what the bug is.

**Comment by laurentS** [comment]
Probably related to #177

**Comment by laurentS** [comment]
This was most likely closed with `2.0.4`
